airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/typing_deduping/BigQueryDestinationHandler.java

```diff
@@ -52,6 +53,11 @@ public Optional<TableDefinition> findExistingTable(final StreamId id) {
     return Optional.ofNullable(table).map(Table::getDefinition);
   }
 
+  @Override
+  public LinkedHashMap<String, TableDefinition> findExistingFinalTables(List<StreamId> streamIds) throws Exception {
```

Comment:
shouldn't this at least delegate to `findExistingTable`? or is this method just unused right now

Reply:
unusued, WIP.